rules/windows/collection_winrar_encryption.toml

```diff
@@ -14,6 +14,40 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Encrypting Files with WinRar or 7z"
+note = """## Triage and analysis
+
+### Investigating Encrypting Files with WinRar or 7z
```

Comment:
```suggestion
### Investigating encrypting files with WinRar or 7z
```

Reply:
Hey @nastasha-solomon we will keep the capitalization matching the rule title, as said in https://github.com/elastic/detection-rules/pull/1799#discussion_r834443948